### ISSUE TITLE
SMS not working

### ISSUE THREAD

**Issue by Kingkhan12398** [WeebSec/PhishX#64]
everything goes fine. email sent good. not going to spam. using smtp2go server. but sms is not working. dont know what the problem is

**Comment by monzroot** [comment]
what? works for you? the url appears?

**Comment by Kingkhan12398** [comment]
yeah its working good. url also appears but only problem is sms not working
